### PR TITLE
Show submission errors in user confirmation form

### PR DESCRIPTION
**Pull request by kristeine**
image

I tried to create a user in staging but got a toast saying "Opprettelse av bruker feilet" which doesn't really explain to the user what's wrong. 

Next step: i18n 😉 

(Also, I added `name` props to some Field elements which were logging console warnings)

### CODE REVIEW

**Comment by finnss** [comment]
The idea of this is nice, but the error messages from the back-end are in english and we want messages the user sees to be in norwegian, no?

**Comment by hanswilw** [comment]
We just discussed that. :) We do already have the english backend errors in other submission errors in master now, we should however add some localization in the future.

File: app/routes/users/components/UserConfirmation.js [hunk 1]
```diff
@@ -98,16 +103,19 @@ const UserConfirmation = ({
           />
           <RadioButtonGroup name="gender">
             <Field
+              name="genderMale"
```
Comment (hanswilw) [inline app/routes/users/components/UserConfirmation.js]: These names are determined by the RadioButtonGroup's `name="gender"` and should be removed or set explicitly to `name="gender"`. They refer to the field `gender` that gets posted to the backend, and redux-form uses the `name` variable to be the same to ensure that the radiobuttons switches correctly.

**Comment by ekmartin** [comment]
I think @odinuge fixed a generic version of this, with `legoForm` - right?

**Comment by odinuge** [comment]
Should work out of the box if 
https://github.com/webkom/lego-webapp/blob/a7b051448592540a08cc199e44a6a824469d4cc6/app/routes/users/components/UserConfirmation.js#L137
is switched to `legoForm` instead

**Comment by ekmartin** [comment]
I'll close this then - wanna send a PR fixing that @odinuge?

**Comment by odinuge** [comment]
Will do

**Pull request closed without merging**